.travis.yml

```diff
@@ -139,6 +139,7 @@ after_success: |
         echo "Building a FULL RELEASE package: $TRAVIS_TAG"
       fi
       python scripts/build_upload.py $TRAVIS_TAG
+      exit $?
```

Comment:
The script failed but the travis job reported success, I believe this should cause the travis job to report failure whenever the script has a non-zero exit.